cmd/benchmark/flood.go

```diff
@@ -30,7 +30,10 @@ import (
 
 func floodTransactions() func(client *wctl.Client) ([]*wctl.TxResponse, error) {
 	return func(client *wctl.Client) ([]*wctl.TxResponse, error) {
-		numWorkers := runtime.NumCPU()
+		numWorkers := runtime.NumCPU() / 2
```

Comment:
It doesn't feel right to simply divide it by 2, as it will return the wrong number of CPU if the processor does not have hyperthreading enabled. I think it's better to have this as a parameter, so that the user can choose how many workers to run for the benchmark (by default it will be `runtime.NumCPU`).